client/my-sites/plugins-wpcom/README.md

```diff
@@ -0,0 +1,81 @@
+# Plugin Page Design
+
+The plugins page provides an overview of the available built-in plugins on WordPress.com and a description of available plugin-like upgrades to help users understand the functionality available to them without installing plugins, and to offer the upgrades for premium functionality.
+
+Because the list of plugins is relatively static, it's probably not worth loading data from an API to populate the list of plugins. Although this could be possible, for performance reasons it's best to declare the data at build-time and simply render it instantly.
+
+That being said, it should also be possible to inject a specific list of plugins either from the API or in testing. This panel will thus be composed of plugin data items that can be expressed declaratively as React components or loaded into the state tree through other means.
+
+## Required state data
+
+```js
+data = {
+  selectedPlugin: // pluginId
+  standardPlugins: [
+    {
+      id: // plugin ID
+      title: // String
+      icon: // Gridicon slug
+      children: // React description of plugin
+    }
+  ],
+  upgradePlugins: [
+    {
+      id: // plugin ID
+      title: // String
+      children: // React description of plugin
+      href: // link to support page on feature
```

Comment:
It would be much better to support this kind of information right within the app, rather than relying on an external source.